### PR TITLE
Reinstate old fallback layout behavior

### PR DESCRIPTION
**Pull request by zenweasel**

### CODE REVIEW

File: client/modules/router/main.js [hunk 1]
```diff
@@ -115,17 +115,31 @@ export function ReactionLayout(options = {}) {
...
-          BlazeLayout.render(layout, layoutToRender);
+          // Look for a layout using the coreLayout and fall back to that
+          Logger.debug("Could not find custom layout, falling back to core");
+          fallbackLayout = shop.layout.reverse().find((x) => selectLayout(x, defaultLayout, workflow));
+          if (!fallbackLayout) {
+            Logger.warn(`Missing layout for ${layout}/${workflow}`);
+            BlazeLayout.render("notFound");
+          }
+        }
+        let newLayoutStructure;
```
Comment (zenweasel) [inline client/modules/router/main.js]: Someone can probably educate me on a better way to do this.

Comment (aaronjudd) [inline client/modules/router/main.js]: just a note on terminology...  I think it's not really a fallbackLayout, it IS the newLayout.

**Comment by aaronjudd** [comment]
👍

**Comment by jshimko** [comment]
👍

**Comment by aaronjudd** [comment]
I'm seeing the index page displaying the login screen, no default grid.  We'll either need to revert or patch this. probably a simple fix.

**Pull request merged**